site/content/docs/desktop/credits.md

```diff
@@ -154,7 +154,7 @@ People who have made contributions to ZAP over the years, in alphabetical order:
 |   | Guifre Ruiz Utges                                                                                                     |
 |   | Dobin Rutishauser - Compass Security AG                                                                               |
 |   | Bunyanuch Saengnet                                                                                                    |
-|   | Najam Ul Saqib ([@njmulsqb](https://www.twitter.com/njmulsqb))                                                        |
+|   | Najam Ul Saqib ([@njmulsqb](https://www.infosec.exchange/@cybersoldier))                                              |
```

Comment:
This file is actually autogenerated, so will be overwritten at the next release.
You need to also update https://github.com/zaproxy/zap-core-help/blob/main/addOns/help/src/main/javahelp/contents/credits.html#L176

Reply:
Alright, is there anyway for me to remove Twitter URL from my page in team section?

Reply:
Of course. For historical reasons thats here https://github.com/zaproxy/zaproxy-website/blob/800767b7d04f604a6dc7c71a7284fab5ed48ba03/site/data/authors.yml#L117

Reply:
Yeah I removed this line and then it was just showing the URL `https://twitter.com` on my profile which is strange.

Reply:
Not in this PR you havnt 😛 
There are currently no changes to the authors file that I can see...

Reply:
Ah, thats a bug in the template then - I'll have a look at that at some point...

Reply:
May I add this as an issue here?

Reply:
Done in #2152.